test/Microsoft.DotNet.MSBuildSdkResolver.Tests/GivenAnMSBuildSdkResolver.cs

```diff
@@ -200,7 +202,9 @@ public void ItReturnsHighestSdkAvailableThatIsCompatibleWithMSBuild(bool disallo
             result.Success.Should().BeTrue($"No error expected. Error encountered: {string.Join(Environment.NewLine, result.Errors ?? new string[] { })}. Mocked Process Path: {environment.ProcessPath}. Mocked Path: {environment.PathEnvironmentVariable}");
             result.Path.Should().Be((disallowPreviews ? compatibleRtm : compatiblePreview).FullName);
             result.AdditionalPaths.Should().BeNull();
-            result.PropertiesToAdd.Should().BeNull();
+            result.PropertiesToAdd.Count.Should().Be(2);
+            result.PropertiesToAdd.Should().ContainKey(_dotnetHost);
+            result.PropertiesToAdd.Should().ContainKey(_msbuildTaskHostRuntimeVersion);
```

Comment:
Validate the `mockRuntimeVersion` value too?

Reply:
thanks! done.